flink-python/pyflink/datastream/connectors/pulsar.py

```diff
@@ -138,10 +138,10 @@ def latest() -> 'StartCursor':
         return StartCursor(JStartCursor.latest())
 
     @staticmethod
-    def from_message_time(timestamp: int) -> 'StartCursor':
+    def from_publish_time(timestamp: int) -> 'StartCursor':
```

Comment:
Should we like this to keep forward-compatibility?

https://github.com/apache/flink/blob/314e276f6c6bff990e82515d6ce90fd6a7c9561d/flink-python/pyflink/datastream/connectors/pulsar.py#L192-L207

Reply:
We have changed the MessgeTime to PublishTime in Java API. This should be a mistake in both Java and Python implementation. So I just introduce this breaking change.

Reply:
Add this breaking change may not sound like a good choice. I add a new method and deprecated the old ones.